Updated LDAP docs (#939)
minio/docs: source/includes/common-minio-external-auth.rst

.. end-minio-ad-ldap-comment

.. start-minio-ad-ldap-console-enable

> #. Log in to the MinIO Console as an admin user, either your root user or one with the ``consoleAdmin`` :ref:`policy <minio-policy>`.

Reviewer: Slight simplification

```suggestion
#. Log in to the MinIO Console as either the :ref:`root <minio-users-root>` user or a MinIO user with the  :userpolicy:`consoleAdmin` policy.
```

#. In the :guilabel:`Identity` section, select :guilabel:`LDAP` and then :guilabel:`Edit Configuration` to configure an Active Directory or LDAP server.
   The minimum required settings are:

   - Server Address
   - Lookup Bind DN
   - Lookup Bind Password
   - User DN Search Base
   - User DN Search Filter

   Not all configuration options are available in the MinIO Console.

>    For additional settings, use the MinIO Client or environment variables.

Author: Couldn't figure out the ideal links (to the other tabs) but I got something.
